### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by butcherpete** [sanack/node-jq#508]
## Description
I have tried `const jq = require('node-jq')` and various means to import node-jq with module set in my package.json.

I keep getting the following error using require('node-jq')

----
/node-jq/node_modules/node-jq/lib/jq.js:1
import exec from './exec';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1026:15)
    at Module._compile (node:internal/modules/cjs/loader:1061:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1149:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/me/Code/node-jq/index.js:3:12)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
----

If I were using node-jq using import instead of require, what's the correct way.
Apologies, there are months of solutions that appear contradictory: specifying module in the package.json, changing file extensions to mjs, etc. I don't know if there is actually an issue or if this is a matter of user error. Thanks again.

### Test Source
I haven't tried to run anything at all, so no code to share.

### Error Message & Stack Trace

```bash
COPY THE ERROR MESSAGE, INCLUDING STACK TRACE HERE
```

## Environment

Tell us which operating system you are using, as well as which versions of Node.js, npm/yarn, and node-jq.

Run the following to get it quickly:
nodejs version:

```bash
Node.js v17.1.0
darwin 19.6.0
```

npm version:

```bash
8.5.5
```

node-jq version:

```bash
  "version": "2.3.2",
```

**Comment by sce9sc** [comment]
Have the same issue . Reverted to 2.0.1 and all works ok

**Comment by nalexiou** [comment]
I reverted to 2.0.1 and I am seeing this error:
` node test.js
events.js:377
      throw er; // Unhandled 'error' event
      ^

Error: spawn /home/pi/node_modules/node-jq/bin/jq EACCES
    at Process.ChildProcess._handle.onexit (internal/child_process.js:274:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:280:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawn /home/pi/node_modules/node-jq/bin/jq',
  path: '/home/pi/node_modules/node-jq/bin/jq',
  spawnargs: [ '.' ]
` 

This is my first installation and trying to get the sample code to run.

**Comment by davesnx** [comment]
Published a new version v2.3.3 with the fix. The work to change babel made so much problems, sorry for the troubles.
